chemprop/cli/hpopt.py

```diff
@@ -238,6 +238,9 @@ def process_hpopt_args(args: Namespace) -> Namespace:
 
 def build_search_space(search_parameters: list[str], train_epochs: int) -> dict:
     if "warmup_epochs" in search_parameters and SEARCH_SPACE.get("warmup_epochs", None) is None:
+        assert (
+            train_epochs >= 6
```

Comment:
`train_epochs > 6`
I ran it for epochs = 6 and it works fine.

Reply:
This is doing what you said exactly though, it only throws error if train_epochs < 6

Reply:
Oh sorry, I misread!
LGTM